src/praisonai-code/praisonai_code/cli/commands/run.py

```
@@ -17,13 +17,19 @@
-        return framework_option_help()
-    except ImportError:
-        return "Framework: praisonai, crewai, autogen"
+        from praisonai_code._wrapper_bridge import optional_wrapper_attr
+
+        framework_option_help = optional_wrapper_attr(
+            "praisonai.framework_adapters.registry", "framework_option_help"
+        )
+        if framework_option_help is not None:
+            return framework_option_help()
+    except Exception:
+        pass
```

Ruff (0.15.20)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)